scss/product/components/_tables.scss

```diff
@@ -10,13 +10,19 @@
   border-radius: $border-radius;
   background-color: $table-bg; // Reset for nesting within parents with `background-color`.
   thead{
-    background: $table-thead-bg-color;
+    th:first-child{
```

Comment:
@Sandeshyc  Can you verify once in design. Seems radius have `border-top-left-radius:4px;` and `border-top-right-radius:4px;`. 

cc: @Pallavi-PH

Reply:
no @is-ashish its 8px
<img width="1152" alt="Screenshot 2020-10-27 at 10 41 07 AM" src="https://user-images.githubusercontent.com/54272277/97259598-2bb2a180-1841-11eb-8ba4-6522c424552a.png">

Reply:
I was checking in generic new table style in figma. There is `border-radius: 4px 4px 0px 0px;` cc: @Pallavi-PH 
![Screenshot from 2020-10-27 12-06-00](https://user-images.githubusercontent.com/28762224/97266082-d761ee80-184d-11eb-94f0-197a34cc8fdf.png)

Reply:
In style guide new table has border radius of `8px 8px 0px 0px` @is-ashish 
<img width="1437" alt="Screenshot 2020-10-27 at 2 38 34 PM" src="https://user-images.githubusercontent.com/34312966/97280373-33cf0900-1862-11eb-9fa7-69a2271dd2be.png">

Reply:
Okay, Border radius of `8px 8px 0px 0px` is fine here. But in first table have `4px 4px 0px 0px`.This is generic de-code table changes so I wanted to confirm here.

Reply:
The first table border radius is diffrent from the other 3 mentioned in style guide. We can ask Victoria which one implement @is-ashish @Sandeshyc